test

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+docker-compose up -d &&
+docker-compose exec app sh ./gradlew test
```

Comment:
correct me, Travis will have to call
./init
./start background
./test
for testing, is that it?
@rayandrews

Reply:
yep, just test it on your PC first before pushing in case misconfiguration @keychera